src/main/java/emissary/transform/HtmlEscapePlace.java

```diff
@@ -92,80 +99,89 @@ public void process(IBaseDataObject d) {
             d.pushCurrentForm(emissary.core.Form.ERROR);
         }
 
+        unescapeAltViews(d);
+        unescapeSummary(d);
+        unescapeDocTitle(d);
+        processEncoding(d);
+        processCurrentForms(d);
+        nukeMyProxies(d);
```

Comment:
Now that these are all testable methods.....

Reply:
Will follow-up once all prs are merged. Ticket is in.